Detectors/ITSMFT/ITS/workflow/include/ITSWorkflow/DCSParserSpec.h

```diff
@@ -142,6 +142,9 @@ class ITSDCSParser : public Task
 
   // ITS Map
   o2::itsmft::ChipMappingITS mp;
+
+  // CCDB url for RCT headers
+  std::string mCcdbUrlRct = "http://alice-ccdb.cern.ch";
```

Comment:
could you set the default to `http://o2-ccdb.internal` ?

Reply:
done!